### PR TITLE
fix: redocly linter

### PR DESCRIPTION
**Pull request by tiagokrebs**
Our second linter is not blocking failing OpenAPI definitions, this PR fixes it 😃 ;
Fix edge-application indentation;

### CODE REVIEW

**Review by maxwelbm — DISMISSED** [review]

File: storageapi.yaml [hunk 1]
```diff
@@ -50,6 +51,7 @@ paths:
   /storage/{version_id}/delete:
     delete:
       operationId: deleteVersion
+      summary: /domains/:version_id
```
Comment (PatrickMenoti) [inline storageapi.yaml:54]: why `domains`?

File: storageapi.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ security:
 paths:
   /storage/{version_id}:
     post:
+      summary: /domains/:version_id
```
Comment (maxwelbm) [inline storageapi.yaml:16]: ?

File: storageapi.yaml (hunk 1 above)
Comment (tiagokrebs) [reply storageapi.yaml:54]: > errroou ~ Silva, Fausto - 2016

File: storageapi.yaml (hunk 2 above)
Comment (tiagokrebs) [reply storageapi.yaml:16]: > errroou ~ Silva, Fausto - 2016

**Review by PatrickMenoti — APPROVED** [review]

**Review by maxwelbm — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Review by PatrickMenoti — COMMENTED** [review]
Approving the changes made by @tiagokrebs 
I need someone else to check the changes I've made

**Review by ftoldato — APPROVED** [review]

**Review by maxwelbm — APPROVED** [review]

**Pull request merged**